--- sphinx-argparse-0.2.5/test/test_parser.py.orig	2017-03-31 10:08:31.000000000 +0200
+++ sphinx-argparse-0.2.5/test/test_parser.py	2020-10-31 10:09:03.610717045 +0100
@@ -1,4 +1,6 @@
 import argparse
+import os
+import sys
 from sphinxarg.parser import parse_parser, parser_navigate
 
 
@@ -140,6 +142,7 @@
     subparser.add_argument('--upgrade', action='store_true', default=False, help='foo2 help')
 
     data = parse_parser(parser)
+    cmd = os.path.basename(sys.argv[0])
 
     assert data['action_groups'][0]['options'] == [
         {
@@ -157,8 +160,8 @@
         {
             'name': 'install',
             'help': 'install help',
-            'usage': 'usage: py.test install [-h] [--upgrade] ref',
-            'bare_usage': 'py.test install [-h] [--upgrade] ref',
+            'usage': 'usage: ' + cmd + ' install [-h] [--upgrade] ref',
+            'bare_usage': cmd + ' install [-h] [--upgrade] ref',
             'action_groups': [
                 {
                     'title': 'Positional Arguments',
@@ -219,12 +222,13 @@
     ]
 
     data2 = parser_navigate(data, 'level1 level2')
+    cmd = os.path.basename(sys.argv[0])
     assert data2['children'] == [
         {
             'name': 'level3',
             'help': '',
-            'usage': 'usage: py.test level1 level2 level3 [-h] foo bar',
-            'bare_usage': 'py.test level1 level2 level3 [-h] foo bar',
+            'usage': 'usage: ' + cmd + ' level1 level2 level3 [-h] foo bar',
+            'bare_usage': cmd + ' level1 level2 level3 [-h] foo bar',
             'action_groups': [
                 {
                     'title': 'Positional Arguments',
